CHANGELOG.md

```diff
@@ -64,18 +65,19 @@
 - [0.10.0](#0100---20170307)
 - [0.9.9 and prior](#099---20170202)
 
+## [3.1.0]
 
-## Unreleased
+> Released 2022/12/02
 
-### Breaking Changes
-
-#### Core
-
-- Change the reponse body for a TRACE method from `The upstream server responded with 405`
-  to `Method not allowed`, make the reponse to show more clearly that Kong do not support
-  TRACE method.
-  [#9448](https://github.com/Kong/kong/pull/9448)
+This major release introduces more flexible management of keys, dynamic log
+level, fixed dataplane ID, HTTP forward proxy support, and other features
+to help ease management. Various plugins are also extended.
+Internal changes are also made to improve Kong's performance.
+As it is a major release, users are advised
```

Comment:
```suggestion
As it is a significant release, users are advised
```

`major` has a specific SemVer connotation, lets avoid it. Also do we have any breaking changes in a minor release?

Reply:
@tyler-ball Could you review the changelog again? I'm not very confident that I understand the changes right.